src/js/components/Hendelser.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { Knapp } from 'nav-frontend-knapper';
+import Informasjon from './hendelser/Informasjon';
+import Config from '../Config';
+import Api from '../Api';
+
+const Hendelser = () => {
+  const [hendelser, setHendelser] = useState([]);
+
+  const getInformasjonHendelser = () => Api
+    .fetchHendelser(Config.dittNav.DITTNAV_HENDELSER_URL)
+    .then((r) => {
+      setHendelser(r);
+    });
+
+  return (
+    <div className="Hendelser">
+      <Knapp onClick={() => (getInformasjonHendelser())}>Hent hendelser</Knapp>
```

Comment:
Legg "Hent hendelser" inn i translations?

Reply:
```suggestion
      <Knapp onClick={() => (getInformasjonHendelser())}><F id="hendelser.hent" /></Knapp>
```

Tenkte du på noe som dette?